app/src/main/java/ch/epfl/polybazaar/SaleDetails.java

```diff
@@ -19,11 +19,14 @@ protected void onCreate(Bundle savedInstanceState) throws NullPointerException{
 
         Bundle b = getIntent().getExtras();
         if(b == null)
-            throw new NullPointerException();
+            b = new Bundle();
```

Comment:
Could you rename that variable with a more explicit name ?

Reply:
> Could you rename that variable with a more explicit name ?

Done